test/spec/extension.js

```diff
@@ -0,0 +1,253 @@
+      var canCopyProperty = zeebeModdleExtension.canCopyProperty(calledElement, extensionElements);
+
+      // then
+      expect(canCopyProperty).to.be.false;
+    });
+  });
+
+
+  describe('<zeebe:LoopCharacteristics>', function() {
+
+    it('should allow on ServiceTask', function() {
+
+      // given
+      var loopCharacteristics = moddle.create('zeebe:LoopCharacteristics'),
+          serviceTask = moddle.create('bpmn:ServiceTask'),
+          extensionElements = moddle.create('bpmn:ExtensionElements');
+
+      extensionElements.$parent = serviceTask;
+
+      // when
+      var canCopyProperty = zeebeModdleExtension.canCopyProperty(loopCharacteristics, extensionElements);
+
+      // then
+      expect(canCopyProperty).not.to.be.false;
+    });
+
+
+    it('should allow on ReceiveTask', function() {
+
+      // given
+      var loopCharacteristics = moddle.create('zeebe:LoopCharacteristics'),
+          receiveTask = moddle.create('bpmn:ReceiveTask'),
+          extensionElements = moddle.create('bpmn:ExtensionElements');
+
+      extensionElements.$parent = receiveTask;
+
+      // when
+      var canCopyProperty = zeebeModdleExtension.canCopyProperty(loopCharacteristics, extensionElements);
+
+      // then
+      expect(canCopyProperty).not.to.be.false;
+    });
+
+
+    it('should allow on SubProcess', function() {
+
+      // given
+      var loopCharacteristics = moddle.create('zeebe:LoopCharacteristics'),
+          subProcess = moddle.create('bpmn:SubProcess'),
+          extensionElements = moddle.create('bpmn:ExtensionElements');
+
+      extensionElements.$parent = subProcess;
+
+      // when
+      var canCopyProperty = zeebeModdleExtension.canCopyProperty(loopCharacteristics, extensionElements);
+
+      // then
+      expect(canCopyProperty).not.to.be.false;
+    });
+
+
+    it('should not allow on CallActivity', function() {
+
+      // given
+      var loopCharacteristics = moddle.create('zeebe:LoopCharacteristics'),
+          callActivity = moddle.create('bpmn:CallActivity'),
+          extensionElements = moddle.create('bpmn:ExtensionElements');
+
+      extensionElements.$parent = callActivity;
+
+      // when
+      var canCopyProperty = zeebeModdleExtension.canCopyProperty(loopCharacteristics, extensionElements);
+
+      // then
+      expect(canCopyProperty).to.be.false;
+    });
+  });
+
+
+  describe('<zeebe:Input>', function() {
+
+    it('should allow on ServiceTask', function() {
+
+      // given
+      var ioMapping = moddle.create('zeebe:IoMapping'),
+          input = moddle.create('zeebe:Input'),
+          serviceTask = moddle.create('bpmn:ServiceTask'),
+          extensionElements = moddle.create('bpmn:ExtensionElements');
+
+      ioMapping.$parent = extensionElements;
+      extensionElements.$parent = serviceTask;
+
+      // when
+      var canCopyProperty = zeebeModdleExtension.canCopyProperty(input, extensionElements);
+
+      // then
+      expect(canCopyProperty).not.to.be.false;
```

Comment:
`to.be.true` is the appropriate shortcut here.

Reply:
I was under the impression that canCopyProperty returned undefined if the copy was allowed.
I haven't tested `to.be.true` because I assumed it wouldn't be.
I'll give it a try tomorrow.